src/users/UserList.jsx

```diff
@@ -78,11 +79,9 @@ export default class UserList extends React.Component {
     return (
       <div style={ [ styles.base, this.props.style ] }>
         <div style={ styles.columnHeader }>
-        {/*  Removed until number is live
           <Heading size="medium">
-            <span style={styles.groupHeader}>{ window.L.t('results') }</span> (106 { window.L.t('users')})
+            <span style={styles.groupHeader}>{ window.L.t('results') }</span> ({this.props.total || 'awaiting pillar #s'} { window.L.t('users')})
```

Comment:
this is more of a feature note. we're awaiting this feature to be built in Pillar.